core/src/components/item/item.ios.scss

```diff
@@ -11,8 +11,18 @@
   --inner-padding-end: #{$item-ios-padding-end};
   --inner-border-width: #{0px 0px $item-ios-border-bottom-width 0px};
   --background: #{$item-ios-background};
-  --background-activated: #000;
-  --background-focused: #000;
+  /**
+   * The active/focus states need
+   * to be the opposite of the background color
+   * so there is enough contrast. The background
+   * defaults to $background-color, so we
+   * default to $text-color which will give us
+   * the opposite color.
+   * i.e. is $background-color is #000
```

Comment:
I am not sure if this comment makes total sense. The `$background-color` and `$text-color` variables don't change, it will only change when the user sets `--ion-background-color` or `--ion-text-color`.